06week/checkpoint2.js

```diff
@@ -0,0 +1,59 @@
+// Given 1000 digits of PI as strings, return an array of the digits as numbers
+const nums = strNums.map(Number);
+// Find the sum of the even values
+const sumEvens = nums.filter((num) => num % 2 === 0).reduce((total, current) => total + current);
+
+
+console.log(sumEvens);
+
+// Return an object with counts for each number
+// {
+//   0: 0, <== increment these counts
+//   1: 0,
+//   2: 0,
+//   ...
+//   8: 0,
+//   9: 0
+// }
+
+const numCount = {
+  0: 0,
+  1: 0,
+  2: 0,
+  3: 0,
+  4: 0,
+  5: 0,
+  6: 0,
+  7: 0,
+  8: 0,
+  9: 0
+};
+
+nums.forEach((item) => {
+  numCount[item] = numCount[item] + 1;
```

Comment:
The final object should look something like this {0:1, 1:4, 2:1, 3:5} where the key is the index and the value is the number in the array.

Answer x.forEach((num, index) =>{ numCount[index] = num })